src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -53,9 +55,11 @@ public PersonCard(Person person, int displayedIndex) {
         this.person = person;
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
-        phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
-        email.setText(person.getEmail().value);
+        birthdate.setText("Age: " + person.getBirthdate().getAge()
+                + "   (d.o.b. " + person.getBirthdate().toDisplayFormat() + ")");
```

Comment:
Given that `Person` holds a reference to their own `Birthdate` object, I think a better use of OOP would be for the `Person` object to calculate its own age and display format rather than here in `PersonCard`. In a way this violates the Law of Demeter and I think it'll be worthwhile to abstract the information in `Person`.

Open for inputs.

Reply:
Yes, definitely agree that it makes more sense to call `Person#getAge()` compared to the current implementation. Will rectify, thanks!